runtime/ftplugin/nroff.vim

```diff
@@ -22,7 +22,19 @@ setlocal comments=:.\\\"
 setlocal sections+=Sh
 setlocal define=.\s*de
 
-let b:undo_ftplugin = 'setlocal commentstring< comments< sections< define<'
+if get(b:, 'preprocs_as_sections')
+  setlocal sections=EQTSPS[\ G1GS
+endif
+
+" XXX 'sections<' does not work
+let b:undo_ftplugin = 'setlocal commentstring< comments< sections& define<'
+
+if get(b:, 'nroff_is_groff')
+  " groff_ms exdented paragraphs are not in the default paragraphs list.
+  setlocal paragraphs+=XP
+  " XXX 'paragraphs<' does not work
+  let b:undo_ftplugin .= ' paragraphs&'
```

Comment:
'paragraphs' and 'sections' do not seem to be local variables, and `let b:undo_ftplugin = 'setl para< sect<'` does not work.

Reply:
yes makes sense.